terraform/monitoring/04_alerting_policies.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Here we create alerting policies on the Service SLO for the frontend service
+# The definition of the SLO can be found in the file "03_service_slo.tf"
+# We alert on budget burn rate, alerting if burn rate exceeds twice the allotted burn rate
+# within the past hour 
+resource "google_monitoring_alert_policy" "frontend_availability_slo_alert" {
+  display_name = "Frontend Service Availability Alert Policy"
+  combiner     = "AND"
```

Comment:
Do you have to set this if there is only one condition?

Reply:
Yes, combiner is a required field.